lightning_pose_app/ui/extract_frames.py

```diff
@@ -446,45 +444,49 @@ def _render_streamlit_fn(state: AppState):
             # write the content of the file to the path, but not while processing
             with open(filepath, "wb") as f:
                 f.write(bytes_data)
-        
+
         # Calculate video duration and frame count
         cap = cv2.VideoCapture(filepath)
         fps = cap.get(cv2.CAP_PROP_FPS)
         frame_count = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
         duration = float(frame_count) / float(fps)
 
-        # Create a formatted string with file size, duration, and frame count
-        #file_info = f"{uploaded_file.name} (Video duration: {duration:.2f} seconds, FPS: {fps},Number of frames: {frame_count} frames)"
-
-        # Display the formatted information
-        #st.write(file_info)
-
         col1.write(uploaded_file.name)
         col2.write(f"{duration:.2f} seconds")
         col3.write(str(frame_count))
 
-        # relese video 
+        # relese video
         cap.release()
-    
+
     # inseret an empty element to create empty space
     st.markdown("###")
 
     col0, col1 = st.columns(2, gap="large")
     with col0:
         # select number of frames to label per video
-        n_frames_per_video = st.text_input("Frames to label per video", 20,help="Specify the desired number of frames for labelling per video. Our system will then intelligently select these frames to maximize the diversity of animal poses captured within each video, optimizing the training process for your model")
+        n_frames_per_video = st.text_input(
+            "Frames to label per video", 20,
+            help="Specify the desired number of frames for labeling per video."
+                 "Our system will then intelligently select these frames to maximize"
+                 "the diversity of animal poses captured within each video,"
+                 "optimizing the training process for your model"
+        )
```

Comment:
make sure to add a space at the end of each line, e.g. 
"...video." -> "...video. "